README.md

```diff
@@ -108,16 +108,16 @@ pub struct HelloWorld;
 // code to automatically bind any exported methods to Godot.
 #[methods]
 impl HelloWorld {
-    
+
     /// The "constructor" of the class.
     fn _init(_owner: Node) -> Self {
         HelloWorld
     }
-    
-    // In order to make a method known to Godot, the #[export] attribute has to be used.
-    // In Godot script-classes do not actually inherit the parent class.
-    // Instead they are"attached" to the parent object, called the "owner".
-    // The owner is passed to every single exposed method.
+
+    // In order to enable access, the owner is passed as the second argument
+    // to every single exposed method. As a result, all exposed methods
+    // MUST have `owner: BaseClass` as their second arguments,
+    // before all other arguments in the signature.
```

Comment:
Thanks for the PR! I feel like you shouldn't have removed the preceding part:

```rust
    // In order to make a method known to Godot, the #[export] attribute has to be used.
    // In Godot script-classes do not actually inherit the parent class.
    // Instead they are"attached" to the parent object, called the "owner".
```

Without the preceding section, all the talk about "owner" does not make much sense. In other words, the full text should be (with a bit of additional rewording and wrapping):

```suggestion
    // To make a method known to Godot, use the #[export] attribute.
    // In Godot, script "classes" do not actually inherit the parent class.
    // Instead, they are "attached" to the parent object, called the "owner".
    //
    // In order to enable access to the owner, it is passed as the second
    // argument to every single exposed method. As a result, all exposed
    // methods MUST have `owner: BaseClass` as their second arguments,
    // before all other arguments in the signature.
```